### PR TITLE
Use @extensionengine/eslint-config

### PR DESCRIPTION
**Pull request by zcuric**
### This PR
- [x] Installs` @extensionengine/eslint-config`
- [x] Lints

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+
```
Comment (kronicker) [inline karma.conf.js]: Extra space

File: client/utils/revision.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ export function getFormatDescription(rev, activity) {
 export function getRevisionAcronym(rev) {
   switch (rev.entity) {
     case 'ACTIVITY':
+      // eslint-disable-next-line no-case-declarations
```
Comment (vladimyr) [inline client/utils/revision.js]: Correct me if I'm wrong but you could just wrap case into a block?
```js
case 'ACTIVITY': {
  const typeArray = rev.state.type.split('_', 2);
  return reduce(typeArray, (acc, val) => acc + val.charAt(0), '');
}
```

File: client/utils/revision.js [hunk 3]
```diff
@@ -75,6 +76,7 @@ export function getRevisionColor(rev) {
   const DEFAULT_COLOR = '#808080';
   switch (rev.entity) {
     case 'ACTIVITY':
+      // eslint-disable-next-line no-case-declarations
```
Comment (vladimyr) [inline client/utils/revision.js]: Same as above :arrow_up:

Comment (zcuric) [reply client/utils/revision.js]: Fixed.

**Comment by zcuric** [comment]
@vladimyr Issues addressed.

**Pull request merged**